client/my-sites/checkout/src/components/checkout-main.tsx

```diff
@@ -400,6 +401,37 @@ export default function CheckoutMain( {
 		checkoutFlow
 	);
 
+	const changeAkismetPro500CartQuantity = useCallback< OnChangeAkProQuantity >(
+		( uuid, productSlug, productId, newQuantity ) => {
+			reduxDispatch(
+				recordTracksEvent( 'calypso_checkout_akismet_pro_quantity_change', {
+					product_slug: productSlug,
+					new_quantity: newQuantity,
+				} )
```

Comment:
Would it be possible to track previous quantity here as well? I am not sure but it seems like it might be useful info

Reply:
Yeah good point! Might be useful info. Will do. 👍